app/_src/quickstart/universal-demo.md

```diff
@@ -198,6 +199,20 @@ mtls:
     - name: ca-1
       type: builtin' | kumactl apply -f -
 ```
+{% endif_version %}
+{% if_version gte:2.9.x %}
+```sh
+echo 'type: Mesh
+name: default
+meshServices:
+  mode: Exclusive
```

Comment:
Have you checked this is all that's needed? I think there's no auto meshservice generator by default in universal

Reply:
It does not look like we need anything according to docs: https://kuma.io/docs/2.9.x/networking/meshservice/#universal Also I check and I see MeshService generated after starting dp